frontend/src/App.tsx

```diff
@@ -64,6 +65,7 @@ const App = () => {
             <PrivateRoute path="/accessdenied" component={AccessDenied}></PrivateRoute>
             <PrivateRoute path="/mapView" component={MapView} />
             <PrivateRoute path="/submitProperty" component={SubmitProperty} />
+            <PrivateRoute path="/edituser" component={EditUserPage} />
```

Comment:
As an FYI, react also supports adding Ids to paths, This is important for pages where it doesn't make sense for a page to be loaded without corresponding data/params. Also important for back/forward navigation. Not strictly necessary but certainly tech debt.